### PR TITLE
[example/flax] add summarization readme

### PR DESCRIPTION
**Pull request by patil-suraj**
# What does this PR do?

This PR adds readme with instructions for the summarization example.

### CODE REVIEW

File: examples/flax/summarization/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+```
+git clone https://huggingface.co/<your-username>/bart-base-xsum
+```
+To ensure that all tensorboard traces will be uploaded correctly, we need to 
+track them. You can run the following command inside your model repo to do so.
+
+```
+cd bart-base-xsum
+
+echo "*.v2 filter=lfs diff=lfs merge=lfs -text" >> .gitattributes
```
Comment (patrickvonplaten) [inline examples/flax/summarization/README.md:33]: After talking to @LysandreJik and @sgugger, we decided that a Python approach is the better solution here. Think you can delete this part and I'll add it to the script :-)

File: examples/flax/summarization/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	--dataset_name="xsum" \
+	--do_train --do_eval --do_predict --predict_with_generate \
+	--num_train_epochs 5 \
+	--learning_rate 5e-5 --warmup_steps 0 \
+	--per_device_train_batch_size 64 \
+	--per_device_eval_batch_size 64 \
+	--overwrite_output_dir \
+	--max_source_length 512 --max_target_length 64 \
+	--push_to_hub
+```
```
Comment (patrickvonplaten) [inline examples/flax/summarization/README.md:63]: Maybe add expected results if possible ? (and even an example link to tensorboard traces maybe?)

**Comment by patrickvonplaten** [comment]
Looks good, maybe also add a `requirements.txt` file?

File: examples/flax/summarization/README.md (hunk 1 above)
Comment (patrickvonplaten) [reply examples/flax/summarization/README.md:33]: The Python solution seemed to require too much work for now given the time constraint. Think for now we can just go with:

```
git lfs track "*tfevents*"
```
like it's written [here](https://github.com/huggingface/transformers/tree/master/examples/flax/language-modeling#masked-language-modeling)

File: examples/flax/summarization/README.md [hunk 3]
```diff
@@ -53,11 +52,15 @@ python run_summarization_flax.py \
...
 	--per_device_train_batch_size 64 \
 	--per_device_eval_batch_size 64 \
 	--overwrite_output_dir \
 	--max_source_length 512 --max_target_length 64 \
 	--push_to_hub
-```
\ No newline at end of file
+```
+
+This should finish in 37min, with validation loss and ROUGE2 of 1.7785 and 17.01 respectively after 6 epochs. raining statistics can be accessed on [tfhub.de](https://tensorboard.dev/experiment/OcPfOIgXRMSJqYB4RdK2tA/#scalars).
```
Comment (patrickvonplaten) [inline examples/flax/summarization/README.md:64]:
```suggestion
This should finish in 37min, with validation loss and ROUGE2 score of 1.7785 and 17.01 respectively after 6 epochs. training statistics can be accessed on [tfhub.de](https://tensorboard.dev/experiment/OcPfOIgXRMSJqYB4RdK2tA/#scalars).
```

**Review by patrickvonplaten — APPROVED** [review]
Feel free to merge when you're ready :-) Thanks for including a test run!
